TestAssets/TestProjects/TestAppWithLibrary/TestLibrary/Helper.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace TestLibrary
+{
+    public static class Helper
```

Comment:
This is just a test asset I am duplicating to have closure of test dependencies inside a single folder.